src/translations/en.json

```
@@ -3433,6 +3433,14 @@
               "sign_in": "Sign in"
             }
           },
+          "llm_task": {
+            "header": "AI Suggestions",
```

This should be sentence-cased.
```suggestion
            "header": "AI suggestions",
```